uwuno/src/test/java/com/learning/uwuno/unit/gameServiceTest.java

```diff
@@ -41,7 +40,7 @@
     final private String newStatus = "Start";
 
     @Test
-    public void testGameService_addRoom() throws Exception {
+    public void testGameService_addRoom() {
```

Comment:
What's the difference between removing the "throws" here vs having them still there inside the playerController test?

Reply:
What do you mean? It was never used

Reply:
Oh I meant since the "throws" was removed here why are they inside the room tests (not the player mis-wrote). Is there a specific reason?

Reply:
Oh the other test file uses functions in the jsonUtils that needs the throw.